step-templates/octopus-create-octoterra-space-s3.json

```diff
@@ -0,0 +1,174 @@
+    "Octopus.Action.Terraform.AdditionalActionParams": "-var=octopus_server=#{OctoterraApply.Octopus.ServerUrl} -var=octopus_apikey=#{OctoterraApply.Octopus.ApiKey} -var=octopus_space_name=#{OctoterraApply.Octopus.Space.NewName} -var=octopus_space_managers=#{OctoterraApply.Octopus.Space.Managers} #{if OctoterraApply.Terraform.AdditionalApplyParams}#{OctoterraApply.Terraform.AdditionalApplyParams}#{/if}",
+    "Octopus.Action.Package.DownloadOnTentacle": "False",
+    "Octopus.Action.RunOnServer": "true",
+    "Octopus.Action.AwsAccount.UseInstanceRole": "False",
+    "Octopus.Action.AwsAccount.Variable": "OctoterraApply.AWS.Account",
+    "Octopus.Action.Aws.AssumeRole": "False",
+    "Octopus.Action.Aws.Region": "#{OctoterraApply.AWS.S3.BucketRegion}",
+    "Octopus.Action.Terraform.TemplateDirectory": "space_creation",
+    "Octopus.Action.AzureAccount.Variable": "a",
+    "Octopus.Action.GoogleCloudAccount.Variable": "a"
+  },
+  "Parameters": [
+    {
+      "Id": "737a32e1-802f-4e15-a67d-cd271aef2c27",
+      "Name": "OctoterraApply.Octopus.Space.NewName",
+      "Label": "Octopus Space Name",
+      "HelpText": "The name of the new space to create. The default value assumes that the step is run against a tenant, and the tenant name matches the space.",
+      "DefaultValue": "#{Octopus.Deployment.Tenant.Name}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "229cc8d4-5200-4f4e-b1c4-8d6464e246a0",
+      "Name": "OctoterraApply.Octopus.Space.Managers",
+      "Label": "Octopus Space Managers",
+      "HelpText": "The name of a team to assign as the space manager.",
+      "DefaultValue": "teams-administrators",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "4d9ad5a5-7058-4f2c-8212-b55f0c7533fd",
+      "Name": "OctoterraApply.Terraform.Workspace.Name",
+      "Label": "Terraform Workspace",
+      "HelpText": "The name of the Terraform workspace. This must be unique for every project this module is deployed to. The default value is based on the space ID that the module is applied to: `#{OctoterraApply.Octopus.SpaceID}`. Leave this as the default value unless you have a specific reason to change it.",
+      "DefaultValue": "#{OctoterraApply.Octopus.Space.NewName  | Replace \"[^A-Za-z0-9]\" \"_\"}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "1e6fc5c3-ab5d-42ec-83e1-0ac7a4b01d31",
+      "Name": "OctoterraApply.Terraform.Package.Id",
+      "Label": "Terraform Module Package",
+      "HelpText": "The package created by [octoterra](https://github.com/OctopusSolutionsEngineering/OctopusTerraformExport). It must include the `space_population` directory.",
```

Comment:
Should this description be updated to the `space_creation` directory as specified by the template directory for the apply parameters?